gradle/plugins/src/main/kotlin/com/saveourtool/save/buildutils/save-demo-agent-configuration.gradle.kts

```
@@ -0,0 +1,35 @@
+/**
+ * Configuration utilities for spring boot projects
+ */
+
+package com.saveourtool.save.buildutils
+
+import de.undercouch.gradle.tasks.download.Download
+import org.gradle.api.tasks.TaskProvider
+import org.gradle.kotlin.dsl.*
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+import java.io.File
+plugins {
+    kotlin("jvm")
+    id("de.undercouch.download")
+}
+
+@Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
+val downloadSaveDemoAgentDistroTaskProvider: TaskProvider<Download> = tasks.register<Download>("downloadSaveDemoAgentDistro") {
+    enabled = findProperty("saveDemoAgentDistroFilepath") != null
+
+    val saveDemoAgentDistroFilepath = findProperty("saveDemoAgentDistroFilepath")?.toString() ?: "file:\\\\not-found"
```

Is the fallback value a `file://` URL?

```suggestion
    val saveDemoAgentDistroFilepath = findProperty("saveDemoAgentDistroFilepath")?.toString() ?: "file://not-found"
```

---

yep, thx